src/logger.js

```diff
@@ -72,12 +72,15 @@ const forceLog = (levelname, message, obj) => {
   }
 };
 
-export const warnClient = msg => {
+export const warnClient = (msg, obj) => {
```

Comment:
```suggestion
export const warnClient = (msg, obj = null) => {
```
If someone doesn't want to pass `obj`.

Reply:
Its will be undefined, its ok